src/ServerScriptService/InstanceComponents/Misc/Sprint.lua

```diff
@@ -0,0 +1,50 @@
+local Players = game:GetService("Players")
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+local Trove = require(ReplicatedStorage.Packages.Trove)
+local Comm = require(ReplicatedStorage.Packages.Comm)
+local WalkSpeedConfig = require(ReplicatedStorage.Common.Config.WalkSpeed)
+
+local Sprint = {}
+Sprint.__index = Sprint
+
+function Sprint.new(character: Model)
+	local serverComm = Comm.ServerComm.new(character, "Sprint")
+	local self = setmetatable({}, Sprint)
+
+	self._character = character
```

Comment:
This should be `self.Instance = character` for consistency. Every component should *probably* be expected to expose an `Instance` field publicly. (2)